tileview/src/main/java/com/qozix/tileview/tiles/TileCanvasViewGroup.java

```diff
@@ -261,9 +263,6 @@ void generateTileBitmap( Tile tile ) {
   }
 
   void addTileToCurrentTileCanvasView( Tile tile ) {
-    if( mTilesAlreadyRendered.contains( tile ) ) {
-      return;
-    }
```

Comment:
hm i don't follow exactly but i believe you.  if you're 100% confident there's no chance of that test ever being true, +1 on this line